notebooks/02-Summary-statistics-and-plots.ipynb

```diff
@@ -0,0 +1,512 @@
+   "metadata": {},
+   "source": [
+    "### Violin Plot of `max_feret` using Seaborn\n",
+    "\n",
+    "Pandas does not have built-in support for Violin Plots so we switch to using Seaborn. Here the `fig` and `ax` objects\n",
+    "are created first and we use the `ax.text()` method to add a string (`text_str`) in a box to the plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68aa386c-fa7a-4ee3-8310-83e4b62d67a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Reset dataframe index to make `image` readily available\n",
+    "df_three_images.reset_index(inplace=True)\n",
+    "fig, ax = plt.subplots(1, 1, figsize=(16, 9))\n",
+    "sns.violinplot(data=df_three_images, x=\"image\", y=\"max_feret\", hue=\"image\", alpha=0.5)\n",
+    "plt.title(\"Minimum Feret\")\n",
+    "plt.ylabel(\"Minimum Feret / nm\")\n",
+    "# Define text for the string to go in a blue text box.\n",
+    "text_str = \"\\n\".join(\n",
+    "    [\"Sodium Concentration    : 0.001mM\", \"Scan Size                        : 200x200\", \"More useful information : :-)\"]\n",
+    ")\n",
+    "props = dict(boxstyle=\"round\", alpha=0.5)\n",
+    "ax.text(\n",
+    "    0.5,\n",
+    "    0.85,\n",
+    "    text_str,\n",
+    "    transform=ax.transAxes,\n",
+    "    fontsize=12,  # verticalalignment=\"top\",\n",
+    "    horizontalalignment=\"center\",\n",
+    "    bbox=props,\n",
+    ")\n",
+    "# Return the index\n",
+    "df_three_images.set_index([\"image\", \"threshold\", \"molecule_number\"], inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f56b8fa-8c55-47a5-9d32-d461a1c639b6",
+   "metadata": {},
+   "source": [
+    "### Joint Plot\n",
+    "[Joint  Plots](https://seaborn.pydata.org/generated/seaborn.jointplot.html) showing the relationship between two variables can be plotted easily."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6aa28c57-66fb-4857-b0df-9ac8f2725173",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.columns\n",
+    "sns.jointplot(data=df, x=\"min_feret\", y=\"max_feret\", hue=\"circular\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "300632da-8be7-4017-a958-f3ebbec01c1d",
+   "metadata": {},
+   "source": [
+    "# Loading Pickles\n",
+    "\n",
+    "One of the features of the `topostats.plotting` module is that it will save the graphs it has generated in Python's own\n",
+    "format for storing objects. These are known as [pickles](https://realpython.com/python-pickle-module/) and typically\n",
+    "have the file extension `.pkl`. After running TopoStats via `run_topostats` and plotting distributions of statistics a\n",
+    "`.pkl` will by default be saved in the output directory. Here we load (\"unpickle\") such a saved file which is typically\n",
+    "located at ~output/summary_distributions/distribution_plots.pkl~ and check its type (its a ~dict~ionary of plots).\n",
```

Comment:
Are these crossed out lines to be in the final notebook?

Reply:
No, its me confusing myself using two different markup languages (Markdown v [Org-mode](https://orgmode.org)). Thanks for picking them up.